pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -1186,6 +1186,11 @@ void recMemcheck(u32 op, u32 bits, bool store)
 	xMOV(edx,eax);
 	xADD(edx,bits/8);
 
+	if ((psxHu32(HW_ICFG) & (1 << 3))) // if we're in PS1 mode adjust ee accordingly.
+	{
+		xADD(edx, (s32)(round(bits / 8.71)));
+	}
+
```

Comment:
This is just doubling it up, you're adding bits /8 then adding another bits/8.71, you need an else condition here.